index.bs

```diff
@@ -8180,28 +8258,27 @@ for the specific requirements that the use of
 
 <div class="example">
 
-    [{{Exposed}}]
-    is intended to be used to control whether interfaces, namespaces, or individual
-    interface or namespace members are available for use only in workers, only in the
-    <code class="idl">Window</code>, or in both.
+    [{{Exposed}}] is intended to be used to control whether interfaces, namespaces,
+    or individual interface or namespace members are available for use only in workers,
+    only in the <code class="idl">Window</code>, or in both.
 
     The following IDL fragment shows how that might be achieved:
 
     <pre highlight="webidl">
-        [PrimaryGlobal]
+        [Exposed=Window, Global=Window]
         interface Window {
           // ...
         };
 
         // By using the same identifier Worker for both SharedWorkerGlobalScope
         // and DedicatedWorkerGlobalScope, both can be addressed in an [Exposed]
         // extended attribute at once.
-        [Global=Worker]
+        [Exposed=Window, Global=Worker]
```

Comment:
Well that works, but that makes no sense. You don't want to expose the global interface object in multiple globals.